extensions/amp-story/1.0/amp-story.js

```diff
@@ -1277,6 +1280,17 @@ export class AmpStory extends AMP.BaseElement {
       return Promise.resolve();
     }
 
+    if (targetPage.isPaywallProtected()) {
```

Comment:
Maybe we should wrap this code in an `isExperiment` block so that we don't break navigation while implementing?

Reply:
Done.